docs/help/account/billing.md

```diff
@@ -0,0 +1,44 @@
+# Billing
+
+The Billing section of your account provides access to the following items:
+
+- information about your current plan
+- your invoice history
+- your billing details
+
+Bump.sh offers monthly or annual billing. Payment is by credit card only; we do not offer any alternatives at this time. We use Stripe to handle all payments, so we never store your bank details.
+
+## Invoices
+
+### Edit Your Billing Information
+
+You can edit the following billing information so that it appears on your upcoming invoices:
+
+- Company name
+- Address
+- VAT Number
+
+:::info
+Edited information will only appear on future invoices; it is not possible to edit the details of an already created invoice.
+:::
+
+### Invoices
+
+Every month, we send your invoice in PDF format to the email address linked to your account. You can specify an alternative email address to which they will be sent, for your accounting department, for example.
```

Comment:
It's not an alternative email, it's an additional one: we sill send the invoice to the user's address.

Reply:
If a billing email is added there, do we send the invoices to both email addresses? (the admin/user one and the billing one?)

Reply:
Yep 👍